cli/src/main.rs

```diff
@@ -494,7 +491,7 @@ fn encode_encrypt<E: Encode>(
 	let worker_api_direct = get_worker_api_direct(matches);
 	let shielding_pubkey: Rsa3072PubKey = match worker_api_direct.get_rsa_pubkey() {
 		Ok(key) => key,
-		Err(err_msg) => return Err(err_msg),
+		Err(err_msg) => return Err(err_msg.to_string()),
```

Comment:
Just out of curiosity, why to string?

Reply:
Because I changed the `worker_api_direct.get_rsa_pubkey()` to return an `Error(ErrorEnum)` instead of an `Error(String)`. To keep the changes minimal, I simply converted it back to a String here. Do you think that's a no go?